data/README.md

```diff
@@ -22,4 +22,41 @@ Registers a `listener` function called everytime the state is updated.
 
 #### `store.dispatch( action: object )`
 
-The dispatch function should be called to trigger the registered reducers function and update the state. An `action` object should be passed to this action. This action is passed to the registered reducers in addition to the previous state.
\ No newline at end of file
+The dispatch function should be called to trigger the registered reducers function and update the state. An `action` object should be passed to this action. This action is passed to the registered reducers in addition to the previous state.
+
+
+### `wp.data.registerSelectors( key: string, selectors: object )`
+
+If your module or plugin needs to expose its state to other modules and plugins, you'll have to register state seclectors.
+
+A selector is function that takes the current state value as a first argument and extra arguments if needed and returns any data extracted from the state.
+
+#### Example:
+
+Let's say the state of our plugin (registered with the key `myPlugin`) has the following shape: `{ title: 'My post title' }`. We can register a `getTitle` selector to make this state value available like so:
+
+```js
+wp.data.registerSelectors( 'myPlugin', { getTitle: ( state ) => state.title } );
+```
+
+#### `wp.data.select( key: string, selectorName: string, ...args )`
```

Comment:
Not sure why, but I see this when in view mode:

![screen shot 2018-01-15 at 10 28 17](https://user-images.githubusercontent.com/699132/34935651-dee5b01e-f9de-11e7-8df7-cb2fc6944da9.png)

Reply:
Fixed, no idea why it even existed :)